config.hh

```diff
@@ -686,6 +686,11 @@ QString getHelpDir() throw();
 QString getOpenCCDir() throw();
 #endif
 
+#ifdef MAKE_ZIM_SUPPORT
+/// Return the path of the MimeTex executable
+QString getTexCgiPath() throw();
```

Comment:
I don't think this public API is needed. How about moving the code into the unnamed namespace in *slob.cc*? Or do you expect *mimetex* to be useful for other dictionary formats as well?